### ISSUE TITLE
Accentuated characters and apostrophes issue in folder's name

### ISSUE THREAD

**Issue by Neomac** [lenalebt/obsidian-pocketbook-cloud-highlight-importer#3]
First of all, thanks a lot for this plugin. Help me to put all my highlights into Obsidian.

When importing them I saw that folder titles have issues with accentuated names.

Here is an example from a book I read:

<img width="288" alt="Capture d’écran 2023-06-14 à 13 59 00" src="https://github.com/lenalebt/obsidian-pocketbook-cloud-highlight-importer/assets/3006818/e126c81b-f652-4147-a667-44b815c96a24">

The title should be: "Père riche, père pauvre", but in the metadata it's all good as you can see here: 

<img width="567" alt="Capture d’écran 2023-06-14 à 13 59 32" src="https://github.com/lenalebt/obsidian-pocketbook-cloud-highlight-importer/assets/3006818/48937c93-0f68-40a5-8ad6-b2f705ee5207">

Also got an issue with apostrophes in book's title like with "Can't Hurt Me Master Your Mind and Defy the Odds":

<img width="289" alt="Capture d’écran 2023-06-14 à 14 09 02" src="https://github.com/lenalebt/obsidian-pocketbook-cloud-highlight-importer/assets/3006818/402f94fc-c193-491a-baec-5286cb59e63f">

Don't know if you use the same parser in the folder creation and in the metadata creation but maybe there is an issue there.

Thanks again, and have a good day.

**Comment by lenalebt** [comment]
Hi @Neomac thanks for your report! I disallowed some (most :upside_down_face: ) characters in the folder names because I did not want to run into trouble with some file systems not allowing some characters, but tbh, it was simply too restrictive. The apostrophes might cause trouble elsewhere, so they will still be disallowed, but it should be somewhat nicer at least with accentuated names (and cyrillic characters, see #4 ).

Have fun, and thanks for the report!
